src/main/java/com/starkbank/Boleto.java

```diff
@@ -378,6 +383,7 @@ public static InputStream pdf(String id) throws Exception {
      * @param id [string]: object unique id. ex: "5656565656565656"
      * @param options [Map of String: Object]: PDF generation options
      * layout [string]: Layout specification. Available options are "default" and "booklet"
+     * hiddenFields [list of strings, default None]: List of string fields to be hidden in Boleto pdf. ex: ["customerAddress"]
```

Comment:
default null e acho que faria bem um *the* entre o hidden in e o Boleto

Reply:
Done